[navigation_links]

# Unrescued Exception

## Example Test

[file_source](../tests/unrescued_exception_test.rb)

Notes:

- Section `Exception not rescued` does not rescue its exception.  The test exits entirely, and any following code in the test is not reached.

## Log

[file_source](../logs/test_unrescued_exception.xml)

Notes:

- The unrescued exception actually is rescued after all, by the log itself.  And of course it's logged.

[navigation_links]
